docs/developer/sql-test-environment/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+# ***************************************************************
+# start local development environment for SQL query testing
+# ***************************************************************
+name: aam-services-sql-environment
+services:
+  db-couch:
+    image: couchdb:3.4
+    volumes:
+      - ~/docker-volumes/aam-digital/db-couch/document-data:/opt/couchdb/data
+    environment:
+      COUCHDB_USER: admin
+      COUCHDB_PASSWORD: docker
+      COUCHDB_SECRET: docker
+    ports:
+      - "5984:5984"
+
+  sqs:
+    image: ghcr.io/aam-digital/aam-sqs-linux:latest
+    #    platform: linux/amd64
+    depends_on:
+      - db-couch
+    ports:
+      - "4984:4984"
+    volumes:
+      - ~/docker-volumes/aam-digital/sqs/data:/data
+    environment:
+      SQS_COUCHDB_URL: http://db-couch:5984
```

Comment:
couldn't this somehow support a public url like dev.aam-digital.com/db/couchdb?

Reply:
theoretically yes, but you will need the credentials for the couchdb instance.

Reply:
if so, maybe we can add this as an option to the docs? Seems easier for query testing than mirroring the db locally (which also requires the admin password or full server access)

Reply:
Yes, i'm already on it.

Reply:
Okay, it's complicated. Looks like the proxy in front of the CouchDB is making trouble...

Reply:
are you trying at `/db` or directly to couch  at `db/couchdb`?

Reply:
`/db/couchdb` - idk why its not working. Just getting some Internal Server Error from SQS, no more information